packages/monaco/src/browser/monaco-keybinding-contexts.ts

```diff
@@ -41,3 +44,32 @@ export class MonacoStrictEditorTextFocusContext extends StrictEditorTextFocusCon
     }
 
 }
+
+/**
+ * The Monaco editor itself is a `textArea` so we have to restrict the default native text input focus context.
+ * It is enabled if the focus is on an `input` or `textArea` and not contained in a Monaco editor.
+ */
+@injectable()
+export class MonacoNativeTextInputFocusContext extends NativeTextInputFocusContext {
+
+    @inject(MonacoEditorProvider)
+    protected readonly editorProvider: MonacoEditorProvider;
+
+    isEnabled(): boolean {
+        return super.isEnabled() && !this.activeElementIsInEditor();
```

Comment:
I think you can do `super.isEnabled() && !this.contextKeyService.match('editorFocus')`

docs for when keys: https://code.visualstudio.com/docs/getstarted/keybindings#_contexts